--- a/doc/Makefile
+++ b/doc/Makefile
@@ -32,7 +32,7 @@ clean:
 	-rm -rf $(BUILDDIR) source/generated
 	-rm generate-stamp
 
-SPHINXEXT_PATH=$(shell python -c 'import sphinx; from distutils.version import LooseVersion; print LooseVersion(sphinx.__version__) < "1.1.2" and "sphinxext" or "/usr/share/pyshared/sphinx/ext"')
+SPHINXEXT_PATH=$(shell python -c 'import sphinx.ext; from distutils.version import LooseVersion; print LooseVersion(sphinx.__version__) < "1.1.2" and "sphinxext" or sphinx.ext.__path__[0]')
 #SPHINXEXT_PATH=sphinxext
 
 generate: generate-stamp
