From: Yaroslav Halchenko <debian@onerussian.com>
Subject: BF(DOC): corrected url to h5py sphinx docs

--- a/doc/source/conf.py
+++ b/doc/source/conf.py
@@ -299,7 +299,7 @@ latex_preamble = r"""
 intersphinx_mapping = {'http://docs.python.org/': None,
                        'http://nipy.sourceforge.net/nipype': None,
                        'http://nipy.sourceforge.net/nipy/stable': None,
-                       'http://h5py.alfven.org/docs': None,
+                       'http://docs.h5py.org/en/2.3/': None,
                        'http://docs.scipy.org/doc/scipy/reference': None,
                        'http://docs.scipy.org/doc/numpy/': None,
                        'http://matplotlib.sourceforge.net/': None,
